### PR TITLE
VZ-11582: Add guide for converting from OAM to Kubernetes

### PR DESCRIPTION
**Pull request by mgianatagh**

### CODE REVIEW

File: content/en/docs/guides/oam-to-kubernetes/_index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: "OAM to Kubernetes Mappings"
+description: "Learn how OAM objects are mapped to Kubernetes objects"
```
Comment (kminder) [inline content/en/docs/guides/oam-to-kubernetes/_index.md:3]: Would be better if object vs resource was consistently used.

File: content/en/docs/guides/oam-to-kubernetes/_index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+weight: 5
+draft: false
+---
+
+### Verrazzano CLI export command
+
+Verrazzano provides a CLI option that you can use to facilitate the migration of an OAM application to be managed as a collection of Kubernetes objects.
+
+The command `vz export oam` will output the YAML for each Kubernetes resource that was generated as a result of deploying an OAM application.  The generated YAML is sanitized so that it can be used to deploy the application.  Fields such as `creationTimestamp`, `resourceVersion`, `uid`, and `status` are not included in the  output.
```
Comment (kminder) [inline content/en/docs/guides/oam-to-kubernetes/_index.md:12]: Extra space between the and output.

File: content/en/docs/guides/oam-to-kubernetes/_index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: "OAM to Kubernetes Mappings"
+description: "Learn how OAM objects are mapped to Kubernetes objects"
+weight: 5
+draft: false
+---
+
+### Verrazzano CLI export command
+
+Verrazzano provides a CLI option that you can use to facilitate the migration of an OAM application to be managed as a collection of Kubernetes objects.
```
Comment (kminder) [inline content/en/docs/guides/oam-to-kubernetes/_index.md:10]: Chould this be command or option?  The term command is use elsewhere.

File: content/en/docs/guides/oam-to-kubernetes/metrics-trait.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+---
+title: "MetricsTrait"
+linkTitle: "MetricsTrait"
+description: "Review the Kubernetes resources Verrazzano creates for an OAM MetricsTrait"
+weight: 5
+draft: false
+---
+
+Verrazzano generates the following Kubernetes resources for a [MetricsTrait]({{< relref "/docs/applications/oam/traits/metrics/metrics.md" >}}):
+* monitoring.coreos.com/v1/ServiceMonitor - A Prometheus custom resource that defines an application from which to scrape metrics.  A ServiceMonitor resource is created by default unless explicitly disabled in the MetricsTrait.
```
Comment (kminder) [inline content/en/docs/guides/oam-to-kubernetes/metrics-trait.md:10]: Should this be "defines a service from which to scrape metrics"?

File: content/en/docs/guides/oam-to-kubernetes/verrazzano-helidon-workload.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+```
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    verrazzanohelidonworkloads.oam.verrazzano.io: d58cca68-b131-47e2-8ea4-3923bff38efa
+  name: hello-helidon-deployment
+  namespace: hello-helidon
+spec:
+  clusterIP: 10.96.254.206
```
Comment (kminder) [inline content/en/docs/guides/oam-to-kubernetes/verrazzano-helidon-workload.md:103]: Might want to make these IP addresses more fake.

File: content/en/docs/guides/oam-to-kubernetes/_index.md (hunk 1 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/_index.md:3]: fixed

File: content/en/docs/guides/oam-to-kubernetes/_index.md (hunk 2 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/_index.md]: fixed

File: content/en/docs/guides/oam-to-kubernetes/_index.md (hunk 3 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/_index.md:10]: fixed - I changed it to command

File: content/en/docs/guides/oam-to-kubernetes/metrics-trait.md (hunk 4 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/metrics-trait.md]: fixed - changed to service

File: content/en/docs/guides/oam-to-kubernetes/verrazzano-helidon-workload.md (hunk 5 above)
Comment (mgianatagh) [reply content/en/docs/guides/oam-to-kubernetes/verrazzano-helidon-workload.md:103]: fixed

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**